apps/backoffice-v2/src/domains/customer/fetchers.ts

```diff
@@ -38,11 +38,13 @@ const CustomerSchema = z.object({
       isMerchantMonitoringEnabled: z.boolean().default(false),
       isExample: z.boolean().default(false),
       isDemo: z.boolean().default(false),
+      hasFullAccess: z.boolean().default(false),
```

Comment:
As mentioned on Slack:
* Make it true by false. We don't want to display the popup by default.
* Rename to something like `showGetFullAccessPopup`/`showTalkToUsPopup`